lib/mix/lib/releases/utils.ex

```diff
@@ -438,4 +438,25 @@ defmodule Mix.Releases.Utils do
       {:unix, _}  -> "\n"
     end
   end
+
+  @doc """
+  Sanitizes text allowing only letters, digits separated by underscore
+
+  ## Examples
+
+      iex> #{__MODULE__}.to_underscore("project.name")
+      "project_name"
+
+      iex> #{__MODULE__}.to_underscore("project name")
+      "project_name"
+
+      iex> #{__MODULE__}.to_underscore("project-name")
+      "project_name"
+  """
+  @spec to_underscore(String.t) :: String.t
+  def to_underscore(text) do
```

Comment:
Might make more sense to name this `sanitize_name`, since the name doesn't really reflect what it does.